Asterix/optics/optical_systems.py

```diff
@@ -376,14 +374,14 @@ def transmission(self, noFPM=True, **kwargs):
         return transmission
 
     def measure_normalization(self):
-        """Functions must me used at the end of all Optical Systems
-        initalization.
+        """
+        Function must me used at the end of all Optical Systems initalization
 
-        Measure 3 different values to normalize the data:
+        Measure 3 important values to normalize the data:
```

Comment:
The "important" doesn't say anything. I would suggest to either keep "different" or just delete this word altogether.